From 8eb43261b2400045769b3eb5ee21d365715cb634 Mon Sep 17 00:00:00 2001
From: Michael W <baddaemon87@gmail.com>
Date: Thu, 25 Jan 2018 12:15:29 +0100
Subject: [PATCH 6/7] WallpaperPicker: Move strings for translation

* AOSP didnt translate these for quite a few languages
* Move to cm_strings (creating our own array as well) to allow people
  to translate these to their languages

Change-Id: I063cfad23e6a05e60f65a61e6618ca015af11593
---
 res/values/cm_strings.xml                              | 10 +++++++++-
 .../wallpaperpicker/WallpaperListBaseAdapter.java      |  6 +++---
 2 files changed, 12 insertions(+), 4 deletions(-)

diff --git a/res/values/cm_strings.xml b/res/values/cm_strings.xml
index d8d98e9..e935329 100644
--- a/res/values/cm_strings.xml
+++ b/res/values/cm_strings.xml
@@ -1,7 +1,8 @@
 <?xml version="1.0" encoding="utf-8"?>
 <!--
 /*
-* Copyright (C) 2017 The LineageOS Project
+* Copyright (C) 2013 The Android Open Source Project
+* Copyright (C) 2017-2018 The LineageOS Project
 *
 * Licensed under the Apache License, Version 2.0 (the "License");
 * you may not use this file except in compliance with the License.
@@ -19,4 +20,11 @@
 
 <resources xmlns:xliff="urn:oasis:names:tc:xliff:document:1.2">
     <string name="no_wallpaper">No wallpaper</string>
+
+    <!-- Option for setting the wallpaper only on the home screen. -->
+    <string name="which_wallpaper_option_home_screen_cm">Home screen</string>
+    <!-- Option for setting the wallpaper only on the lock screen. -->
+    <string name="which_wallpaper_option_lock_screen_cm">Lock screen</string>
+    <!-- Option for setting the wallpaper on both the home screen and lock screen. -->
+    <string name="which_wallpaper_option_home_screen_and_lock_screen_cm">Home screen and lock screen</string>
 </resources>
diff --git a/src/com/android/wallpaperpicker/WallpaperListBaseAdapter.java b/src/com/android/wallpaperpicker/WallpaperListBaseAdapter.java
index 2a6b53b..4b8c7b7 100644
--- a/src/com/android/wallpaperpicker/WallpaperListBaseAdapter.java
+++ b/src/com/android/wallpaperpicker/WallpaperListBaseAdapter.java
@@ -33,10 +33,10 @@ public class WallpaperListBaseAdapter extends BaseAdapter {
     private final Context context;
 
     private static Pair<Integer, Integer>[] DIALOG_ENTRIES = new Pair[] {
-        new Pair(R.drawable.ic_home_screen, R.string.which_wallpaper_option_home_screen),
-        new Pair(R.drawable.ic_lock_screen, R.string.which_wallpaper_option_lock_screen),
+        new Pair(R.drawable.ic_home_screen, R.string.which_wallpaper_option_home_screen_cm),
+        new Pair(R.drawable.ic_lock_screen, R.string.which_wallpaper_option_lock_screen_cm),
         new Pair(R.drawable.ic_home_screen_and_lock_screen,
-                R.string.which_wallpaper_option_home_screen_and_lock_screen)
+                R.string.which_wallpaper_option_home_screen_and_lock_screen_cm)
     };
 
     private class ViewHolder {
-- 
2.17.1

